exonum-node/src/schema.rs

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use exonum::{
+    crypto::PublicKey,
+    helpers::Round,
+    merkledb::{
+        access::{Access, AccessExt, RawAccessMut},
+        ListIndex, MapIndex,
+    },
+    messages::Verified,
+};
+
+use std::iter;
+
+use crate::messages::{Connect, Message};
+
+const CONSENSUS_MESSAGES_CACHE: &str = "core.consensus_messages_cache";
+const CONSENSUS_ROUND: &str = "core.consensus_round";
+const PEERS_CACHE: &str = "core.peers_cache";
+
+/// Schema for an Exonum node.
+#[derive(Debug)]
+pub(crate) struct NodeSchema<T> {
+    access: T,
```

Comment:
There are only 3 indexes in this schema, so I think using a declarative schema layout with eager initialization won't harm the performance.